core/tests/test_utils.py

```diff
@@ -511,15 +512,15 @@ def save_new_valid_collection(
             exploration_id='an_exploration_id',
             end_state_name=DEFAULT_END_STATE_NAME):
 
-        """Creates a collection and save exploration details in it.
+        """ Creates a collection and save exploration details in it.
 
         Args:
             collection_id: int. Id for the newly created collection.
             owner_id: int. Owner id for the newly created collection.
 
         Returns:
-            Newly created collection containing corresponding exploration
-            details.
+            Collection. Newly created collection containing corresponding
+            exploration details.
```

Comment:
@zhengxit It needs to be four spaces indented after the line above it. Is that clear?